data_structures/stacks/__init__.py

```diff
@@ -1,22 +0,0 @@
-class Stack:
```

Comment:
Not sure if this is okay (any other files are using this?).

Reply:
This file should be empty. No other files are using this

Reply:
Do the other files all have tests so that we know that is true?

Reply:
Anything need to be updated ?

Reply:
seems like there's another stack inside stack/

Reply:
Building is passed. So I think we should clear content in __init__.py